docs/_pages/faq.md

```diff
@@ -96,23 +96,42 @@ server {
 ```
 
 ## How to keep map data after reboot
-In default the robot will lose map data after each reboot. To prevent the map reset use the mirobo command line tool (same tool you use for the installation process) and execute `mirobo raw-command set_lab_status 1`.
-This can be also done via filesystem access `echo -n "1" > /mnt/data/rockrobo/lab.cfg` on the robot itself.
+
+By default, the robot will lose map data after each reboot. For Gen2 devices,
+to prevent the map reset, use the `mirobo` command line tool (same tool you use
```

Comment:
This is also integrated in valetudo since quite a while

Settings >Persistent Data

![image](https://user-images.githubusercontent.com/974410/82360833-423f0c00-9a0a-11ea-86bb-bbcd6d32c1b8.png)